docs/src/main/paradox/connectors.md

```diff
@@ -8,6 +8,7 @@
 * [Apache Geode](geode.md)
 * [Apache Kafka](kafka.md)
 * [Apache Kudu](kudu.md)
+* [Apache Parquet](avroparquet.md)
```

Comment:
Was it intentional to use "Apache" instead of "Avro"? Both are Apache projects, but it would better to follow the module name.